app/controllers/posts_controller.rb

```diff
@@ -33,32 +33,32 @@ def create
       render :new
     end
   end
-  
+
   def edit
     @post = Post.find(params[:id])
     @community = Community.find(params[:community_id])
```

Comment:
see it is used here as well, community find

Reply:
resolved